python_scripts/tests/api_tests/database_api/get_config.json.pat

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
Is there any way to programmatically generate this file? A lot of this does not change, but if anything does, it is tedious to make the change in N different places where the data is duplicated.

Reply:
Pattern file can be easily produced by renaming the output file which testing framework generates. Of course it needs to be verified and accepted by test maintainer.